src/main/webapp/css/bootstrap-theme.min.css

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
Minified front-end files should be treated like Java .class files (i.e., they shouldn't be under VCS). You might consider looking at something like Webpack or Gulp to generate these files on the fly.